projects/plugins/jetpack/tests/php/json-api/test-class.wpcom-json-api-list-users-endpoint.php

```diff
@@ -0,0 +1,164 @@
+<?php
+/**
+ * Jetpack WPCOM JSON API `sites/%s/users` endpoint unit tests.
```

Comment:
I added these basic tests because they weren't there.

Testing the viewer role might have to be done on WPCOM as it's gated behind the `IS_WPCOM` constant and calls internal methods, e.g., get_private_blog_users